src/AVPRClient/Extensions.cs

```diff
@@ -90,5 +90,54 @@ DateTimeOffset releaseDate
                 };
             }
         }
+
+        public static AVPRIndex.Domain.Author [] toAuthors (
```

Comment:
I think `MapAuthors` is a better name here

Reply:
Or `GetAuthors`. Idk. Whatever you like best of these 2.

Reply:
i chose GetAuthors, i think it's more clear